.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,44 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fabric? This template includes necessary information to get started, and steps to completion
+
+---
+<!-- Use this template for new components or new component varients -->
+
+## Component Details
+
+#### Exports
+<!-- What will be included in index.ts file -->
+
+#### Imports
+<!-- What other components/modules will your component be using -->
+
+#### Intended Package
+<!-- OUFR/New Package/Charting etc... -->
```

Comment:
If this is a prototype component, start in `@uifabric/experiments`. If you feel that there is a new package required, please indicate the requested name here.